tests/integration/bigquery.spec.ts

```diff
@@ -143,6 +143,25 @@ suite("@dataform/integration/bigquery", ({ after }) => {
 
     // run cache assertions
 
+    const testTarget = {
+      schema: 'df_integration_test',
+      name: 'depends_on_example_view',
+      database: 'dataform-integration-tests'
+    }
+    compiledGraph.tables.push({
```

Comment:
can you just add this as a file instead of inserting it into the compilation graph? otherwise this is bound to get out of sync with the compilation logic.